gerrit/changes/change.py

```diff
@@ -0,0 +1,124 @@
+"""
+Change
+======
+
+Manage gerrit changes
+"""
+
+from gerrit.helper import decode_json
+from gerrit.error import UnhandledError
+from gerrit.projects.project import Project
+
+def add_change(gerrit_con, project, subject, branch, options):
+    """
+    Create a change
+    :param gerrit_con: Gerrit connection
+    :type gerrit_con: gerrit.Gerrit
+    :param project: Project to create change in
+    :type project: str or gerrit.project.Project
+    :param subject: Subject of the change
+    :type subject: str
+    :param branch: The name of the target branch
+    :type branch: str
+    :param options: Additional options
+    :type options: Dict
+    """
+
+    r_endpoint = "/a/changes/"
+
+    if isinstance(project, Project):
+        project = project.name
+
+    data = {
+        'project': project,
+        'subject': subject,
+        'branch': branch,
+    }
+
+    if options is None:
+        options = {}
+
+    for key in options.keys():
+        data[key] = options[key]
+
+    req = gerrit_con.call(
+        request='post',
+        r_endpoint=r_endpoint,
+        r_payload=data,
+    )
+
+    result = req.content.decode('utf-8')
+
+    if req.status_code == 201:
+        return Change(gerrit_con,
+                      project,
+                      branch,
+                      decode_json(result).get('change_id')
+                     )
+    else:
+        raise UnhandledError(result)
+
+class Change(object):
+    """Manage gerrit changes"""
+
+    def __init__(self, gerrit_con, project, branch, change_id):
+        """
+        :param project: Project that contains change
+        :type project: str or gerrit.projects.Project
+        :param change_id: ID of change
+        :type change_id: str
+        :param branch: Branch change exists in
+        :type branch: str
+        """
+
+        if isinstance(project, Project):
+            project = project.name
+
+        if project == '':
```

Comment:
Perhaps better to say if not project:

empty string will resolve to False as well